tests/update_workflow.go

```diff
@@ -3542,6 +3542,8 @@ func (s *FunctionalSuite) TestUpdateWorkflow_ScheduledSpeculativeWorkflowTask_Lo
 }
 
 func (s *FunctionalSuite) TestUpdateWorkflow_StartedSpeculativeWorkflowTask_LostBecauseOfShardMove() {
+	disableImmediateUpdateAbortOnShardMove(s)
```

Comment:
This and other tests are asserting the behavior of a "lost Update" (ie one that isn't actively aborted on shard movements). Therefore, the functionality is disabled for these.